### PR TITLE
[BUGFIX RELEASE] [BUGFIX LTS] Do not ignore dotfiles in ESLint

### PR DESCRIPTION
**Pull request by Gaurav0**
For #8070 

Also related to https://github.com/ember-cli/rfcs/pull/121

Edit: added a test for this here: https://github.com/ember-cli/broccoli-lint-eslint/pull/139

### CODE REVIEW

File: blueprints/app/files/.eslintignore [hunk 1]
```diff
@@ -16,3 +16,5 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+!.*
```
Comment (kellyselden) [inline blueprints/app/files/.eslintignore]: Can you put this under miscellaneous please

Comment (Gaurav0) [reply blueprints/app/files/.eslintignore]: Yes.

**Comment by rwjblue** [comment]
I think this should pass if you rebase...

**Comment by Gaurav0** [comment]
@rwjblue Still doesn't pass.

**Comment by Turbo87** [comment]
@Gaurav0 can you target the `release` branch directly with this?

**Comment by rwjblue** [comment]
The remaining CI failure is addressed in #8116

**Comment by rwjblue** [comment]
@Turbo87 - Lets just shoot for `beta`. I'd hate to do a 3.4 patch release that is "breaking" to folks that have linting errors in their dotfiles...

**Comment by Turbo87** [comment]
@rwjblue yeah, makes sense. I just went for `release` because the PR title mentioned that branch.

**Pull request closed without merging**